stock_production_lot_firmware_version/views/stock_view.xml

```diff
@@ -0,0 +1,29 @@
+<?xml version="1.0"?>
+<odoo>
+    <!-- Lots/Serial Numbers form view -->
+    <record id="view_production_lot_form_fm_version_view" model="ir.ui.view">
+        <field name="name">stock.production.lot.form</field>
+        <field name="model">stock.production.lot</field>
+        <field name="inherit_id" ref="stock.view_production_lot_form"/>
+        <field name="arch" type="xml">
+            <data>
+                <field name="ref" position="after">
+                    <field name="firmware_version"/>
+                </field>
+            </data>
+        </field>
+    </record>
+    <!-- Lots/Serial Numbers search view -->
+    <record id="search_product_lot_filter_fm_version" model="ir.ui.view">
+        <field name="name">Production Lots Filter</field>
+        <field name="model">stock.production.lot</field>
+        <field name="inherit_id" ref="stock.search_product_lot_filter"/>
+        <field name="arch" type="xml">
+            <data>
+                <field name="name" position="attributes">
+                    <attribute name="filter_domain">['|','|',('name','ilike',self),('ref','ilike',self),('firmware_version','ilike',self)]</attribute>
```

Comment:
As soon as another addon will have the same idea about adding a field in this domain, only one of them will work. Wouldn't it be safer to add a new search filter? Or maybe customize the name_search?

Reply:
Agree.

We have two options for these.

1. Adding new firmware version filter. (User need to select particular these filter option)

2. Override `name_search`. (only append domain filter)

What you suggestion ?

Reply:
@bodedra As long as the user can search using the firmware version field in the advanced filters, we are good. No need for a specific filter, nor include it in the name_search.